checkov/terraform/checks/resource/yandexcloud/VPCSecurityGroupAllowAll.py

```diff
@@ -0,0 +1,31 @@
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckResult, CheckCategories
+
+class VPCSecurityGroupAllowAll(BaseResourceCheck):
+    def __init__(self) -> None:
+        name = "Ensure security group does not contain allow-all rules."
+        id = "CKV_YC_19"
+        categories = [CheckCategories.GENERAL_SECURITY]
+        supported_resources = ["yandex_vpc_security_group"]
+        super().__init__(
+            name=name,
+            id=id,
+            categories=categories,
+            supported_resources=supported_resources,
+        )
+
+    def scan_resource_conf(self, conf):
+        if 'ingress' in conf.keys():
+            cidr_block = conf['ingress'][0]['v4_cidr_blocks']
+            if cidr_block[0][0] == "0.0.0.0/0":
```

Comment:
I think you need to iterate over a possible list of IP ranges or is it not possible to have other IP ranges, when you have `0.0.0.0/0` inside?

Reply:
You are very much right, fixed this block.